fawltydeps/settings.py

```diff
@@ -215,9 +216,6 @@ def create(cls, cmdline_args: argparse.Namespace) -> "Settings":
         if {"verbose", "quiet"}.intersection(args_dict.keys()):
             ret["verbosity"] = args_dict.get("verbose", 0) - args_dict.get("quiet", 0)
 
-        if "custom_mapping" in args_dict.keys():
-            ret["custom_mapping"] = args_dict.get("custom_mapping")
```

Comment:
Assuming this should be `custom-mapping-path` not `custom-mapping`, shouldn't setting this option in the CLI override what has been configured via other means. Therefore, shouldn't these lines stay?

Reply:
It by default overrides configuration file. This is not needed, in fact was an artifact in the previous PR.